src/common_tasks/corpus_candidate_set.py

```diff
@@ -29,7 +29,7 @@ def validate_candidate_items(corpus_items: List[Dict]):
     expected_keys = ['ID', 'PUBLISHER']
 
     assert len(corpus_items) >= min_item_count
-    assert all(list(corpus_item.keys()) == expected_keys for corpus_item in corpus_items)
```

Comment:
Nit: This function should be moved to `api_clients.sqs`. It doesn't operate on corpus candidates. Can be done in a future PR.